webrtc-stats.html

```diff
@@ -1098,6 +1099,15 @@ <h2>
                   you the number of delta frames decoded.
                 </p>
               </dd>
+              <dt>
+                <dfn>framesRendered</dfn>
+              </dt>
+              <dd>
+                <p>
+                  Only [= map/exist =]s for video. It represents the total number of frames that have been
+                  rendered. It is incremented just after a frame has been rendered.
```

Comment:
I think it would be good to link to a definition of "rendered" - this may have to go into the terminology section.

Reply:
I agree that this should be clarified, but because this language is already used in several places, I filed #718 as a follow-up to make sure we get all of them in one issue